changelog/fragments/remove-exec-entrypoint.yaml

```diff
@@ -0,0 +1,6 @@
+entries:
+  - description: >
+      The `operator-sdk` command no longer supports `exec-entrypoint` subcommand for ansible and helm operators. The entrypoints
+      for ansible/helm base images are from separate binaries present inside `cmd/` folder.
+    kind: "removal"
+    breaking: false
```

Comment:
The changelog generator will not allow `removal` and `breaking: false`.

Now that I think about this more, I don't think we actually need a changelog fragment because this is an internal change that is not user-facing.